src/.pre-commit-config.yaml

```diff
@@ -25,5 +25,5 @@ repos:
         # supported by your project here, or alternatively use
         # pre-commit's default_language_version, see
         # https://pre-commit.com/#top_level-default_language_version
-        language_version: python3.8
+        language_version: python3.9
```

Comment:
hope this is ok, couldnt commit otherwise and seems the project supports 3.9 and thus is correct based on the above comment